js/status.js

```diff
@@ -0,0 +1,234 @@
+(function ($, window)
+{
+  // register namespace
+  $.extend(true, window, {
+    "StatusApp": StatusApp
+    }
+  );
+
+  /**
+   * This application builds and displays a status table constructed from
+   * queries to a list of services. Each row in the table contains
+   * the status of a service.
+   *
+   * param: _options - values to be supplied to this application
+   */
+  function StatusApp(_options)
+  {
+    const VERSION_PARAM = "?version=" + new Date().getTime();
+    const XML_CONTENT_TYPE = 'application/xml; charset=utf-8';
+    const TEXT_CONTENT_TYPE = 'text/plain; charset=utf-8';
+    const STANDARD_ID = '[standardID="ivo://ivoa.net/std/VOSI#availability"]';
+    const CAPS_SERVERS = ["www.canfar.phys.uvic.ca", "www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca"];
+    const REFRESH_PERIOD = 180000; // 180 seconds or 3 minutes
+    const IMAGES_DIR = _options.images;
+    const RESOURCE_CAPS_URL = _options.resourceCaps;
+
+    // serviceKVPs contains key/value pairs which map service capabilities URL to 
```

Comment:
This is a comment with an example element and not commented out code. However I shall change "//" to "/*".